streaming/src/test/scala/org/apache/spark/streaming/CheckpointSuite.scala

```diff
@@ -640,12 +640,14 @@ class CheckpointSuite extends TestSuiteBase with DStreamCheckpointTester
         val fileStream = ssc.textFileStream(testDir.toString)
         // Make value 3 take a large time to process, to ensure that the driver
         // shuts down in the middle of processing the 3rd batch
-        CheckpointSuite.batchThreeShouldBlockIndefinitely = true
+        CheckpointSuite.batchThreeShouldBlockALongTime = true
         val mappedStream = fileStream.map(s => {
```

Comment:
Could we maybe make this as below?
```scala
fileStream.map { s =>
  ...
}
```

This was discussed in http://apache-spark-developers-list.1001551.n3.nabble.com/Coding-style-question-about-extra-anonymous-closure-within-functional-transformations-td17125.html.